src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,67 @@
+package study;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 사이즈가 동일하면 성공한다.")
+    void shouldSuccessSizeIsSame(){
+        Assertions.assertEquals(numbers.size(), 3);
+    }
+
+    @Test
+    @DisplayName("contains() 메소드를 활용해 1, 2, 3의 값이 존재하는지를 확인한다")
+    void contains() {
+        assertThat(numbers.contains(1)).isTrue();
```

Comment:
```suggestion
        assertThat(numbers).contains(1);
```

collection의 경우 위와 같이 구현도 가능함